client/src/app.jsx

```diff
@@ -37,13 +37,14 @@ class PhotoCarousel extends React.Component {
   }
 
   componentDidMount() {
-    this.getAllPhotos();
+    this.getAllPhotos(this.urlProductId());
   }
 
-  getAllPhotos() {
+  getAllPhotos(id) {
     $.ajax({
       url: 'http://localhost:3001/api/photos',
       type: 'GET',
+      data: { product_id: id },
       success: (photos) => {
         this.setState({
           mainPic: photos[0],
```

Comment:
Is the photos[0] correct or should it be just photos?

Reply:
Photos[0] is correct here. Setting state for the single display photo versus the array oh photos!